recordedit/form.controller.js

```diff
@@ -7,7 +7,7 @@
         var vm = this;
         var context = $rootScope.context;
         vm.recordEditModel = recordEditModel;
-        vm.editMode = context.filter || false;
+        vm.editMode = (context.filter && !context.copy) || false;
```

Comment:
Or you may elect to keep the `|| false` if you feel it's more readable. I'm ambivalent.